draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -145,17 +145,20 @@ using multiple and variable time windows and dynamic quotas, or implementing con
 
 The goals of this proposal are:
 
-   1. Standardizing the names and semantics of rate-limit headers
-      to ease their enforcement and adoption;
+  Interoperability:
+  : Standardization of the names and semantics of rate-limit headers
+    to ease their enforcement and adoption;
```

Comment:
Does this look good in the final form of the document? Rendering it in markdown it is all shown in the same line with two colons next to each other.

Reply:
@unleashed this is the intended output https://i.imgur.com/XPXk0pB.png

If you prefer it on two lines, we can add a `{: vspace="0"}` WDYT?

Reply:
It looks ok then, good with me!